<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDefectDialog</class>
 <widget class="QDialog" name="EditDefectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Defect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="defectListComboBox"/>
   </item>
   <item>
    <widget class="QLineEdit" name="defectNameLineEdit">
     <property name="placeholderText">
      <string>Defect Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="submitButton">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
